drivers/input/input_gt911.c

```diff
@@ -130,25 +129,31 @@ static int gt911_process(const struct device *dev)
 		return r;
 	}
 
-	/* obtain first point X, Y coordinates and event from:
-	 * REG_P1_XH, REG_P1_XL, REG_P1_YH, REG_P1_YL.
+	/*
+	 * obtain point X, Y coordinates and event
 	 */
-	reg_addr = REG_FIRST_POINT;
-	r = gt911_i2c_write_read(dev, &reg_addr, sizeof(reg_addr), &pointRegs, sizeof(pointRegs));
-	if (r < 0) {
-		return r;
-	}
+	if (points) {
+		if (points > 1) {
+			input_report_abs(dev, INPUT_ABS_MT_SLOT, points, true, K_FOREVER);
+		}
+
+		for (int i = 0; i < points; i++) {
```

Comment:
`uint8_t i`, save few bytes on the stack

Reply:
Done, thanks. :)